### DISCUSSION TITLE
How to debug the rejected deploy request

### DISCUSSION THREAD

**Discussion by muhaimincs** [planetscale/discussion#62] [Q&A] [answered]
I encountered this: `error loading logical schema: found 1 failed statements in the schema`. Where do I find the failed statement in schema.

Thanks

**Comment by lkburns** [comment, marked as answer]
Hey @muhaimincs, thanks for giving PlanetScale a try! If you don't mind sharing the name of your organization and the database where you're receiving the error, we can take a look and see what might be causing the issue. Also, feel free to file a [support ticket here](https://www.planetscale.com/support), if you'd prefer to share that info privately.

**Reply by muhaimincs** [reply]
here is my details mcs-tech / usahawan-db. Please inspect the deploy request from branch `dev`

**Reply by jgreet** [reply]
Sorry for the confusion and the unhelpful error message. We're working on fixing that.

It looks like the problem with your deploy request is that it's trying to create a column named `updated_at` with a default value of `000-00-00 00:00:00`. We have `NO_ZERO_DATE` set for PlanetScales' underlying MySQL servers which makes this default invalid. You may want to look at using NULL or `1970-01-01 00:00:01` instead.

See also: https://dev.mysql.com/doc/refman/8.0/en/datetime.html

If using a different default doesn't work for you please [open a support ticket](https://www.planetscale.com/support) and let us know why. Thanks!
